src/dali/logger.py

```diff
@@ -15,6 +15,10 @@
 
 import logging
 
+import progressbar
```

Comment:
Can you import directly the symbol you're going to use (wrap_stderr)?

Reply:
No, actually - `progressbar.streams` is an object rather than a module: https://github.com/wolph/python-progressbar/blob/develop/progressbar/utils.py#L531

I've made the import as granular as feasible in the most recent push.